src/overview/tabs/History/cards/MigrationRow.tsx

```diff
@@ -0,0 +1,81 @@
+import type { FC } from 'react';
+import type { RowProps } from 'src/components/common/TableView/types';
+import { ModalHOC } from 'src/modules/Providers/modals/ModalHOC/ModalHOC';
+import { TableLinkCell } from 'src/modules/Providers/utils/components/TableCell/TableLinkCell';
+import VisibleTableData from 'src/modules/Providers/utils/components/TableCell/VisibleTableData';
+import { getMigrationVMsStatusCounts } from 'src/plans/details/components/PlanStatus/utils/utils';
+import VMStatusIconsRow from 'src/plans/details/components/PlanStatus/VMStatusIconsRow';
+import { getMigrationStatusLabel } from 'src/plans/details/tabs/Details/components/MigrationsSection/components/utils/utils';
+
+import { ConsoleTimestamp } from '@components/ConsoleTimestamp/ConsoleTimestamp';
+import {
+  MigrationModelGroupVersionKind,
+  PlanModelGroupVersionKind,
+  type V1beta1Migration,
+  type V1beta1Plan,
+} from '@kubev2v/types';
+import { Split } from '@patternfly/react-core';
+import { Tr } from '@patternfly/react-table';
+import { getName, getNamespace } from '@utils/crds/common/selectors';
+
+type MigrationRowProps = RowProps<V1beta1Migration> & {
+  plans: V1beta1Plan[];
+};
+
+const MigrationRow: FC<MigrationRowProps> = ({
+  plans,
+  resourceData: migration,
+  resourceFields,
+}) => {
+  const plan = plans.find((pl) => {
+    return (
+      pl.metadata?.name === migration.spec?.plan?.name &&
+      pl.metadata?.namespace === migration.spec?.plan?.namespace
```

Comment:
you can also compare by uid, this would give you just one check :) 
please use getUID(plan) and create a similar selector to get from a migration it's plan UID

Reply:
I can add uid check, is it enough or maybe leave name+namespace as fallback check?
Also I'm not sure about creating selectors for other CRDs/objects, that can get pretty involved I think :)